### DISCUSSION TITLE
How view contacts

### DISCUSSION THREAD

**Discussion by mrma13** [hmatuschek/qdmr#356] [General]
When I click on the Contact Lists the info shown seems to be Talk Groups. 
How do I view the actual list of contacts?
Is it possible to upload/download contact lists?
Anytone D878UV, FW 3.02, Hardware v1.0
Thanks

**Comment by hmatuschek** [comment]
In fact, AnyTone labels them wrong. What they call "talk groups" are just DMR contacts (including private calls) and what they call "contacts" is just a call-sign database that is used to translate DMR IDs to names and calls for displaying. You can upload a call-sign DB to the radio using the "Upload call-sign DB" button in the tool bar. See also https://dm3mat.darc.de/qdmr/manual/ch02s11.html#idp21 on how to control, which call-signs are written to the device.

**Comment by mrma13** [comment]
Is it possible to DL the call-sign database from the radio and save it? I did not see this in the provided reference.

Thanks for the reply and info.

**Reply by hmatuschek** [reply]
No, it is not. As qdmr curates the call-sign DB itself, there is no need to save it. Also, qdmr updates the list of all DMR ID once every week. So, writing a fresh one from time to time, will keep your call-sign DB up to date.

**Comment by allesand** [comment]
The way I use it: qdmr "Contacts"/Anytone "Talk Groups" contain hand-edited Private Calls and Group Calls so I can use those by hitting PTT on the selected entry (I do not use the "one channel entry per talkgroup" concept). In qdmr you can select Private Call or Group call and qdmr looks up the DMR ID of the callsign or the talkgroup, makes it easy to edit that list. 
(Also, the Anytone displays info from that list when not in callsign DB)
The "callsign DB" on the other hand is just pushed to the radio as above - more info on the heard stations that way.
If you want the complete list as a "readable" file, there is a download on radioid.net ?
